### ISSUE TITLE
plugin-apply-shorthand-properties: ignore imports

### ISSUE THREAD

**Issue by EvgenyOrekhov** [coderaiser/putout#24]
Currently, plugin-apply-shorthand-properties transforms this code

```js
import evolve from "ramda/src/evolve";
import not from "ramda/src/not";
import mergeDeepRight from "ramda/src/mergeDeepRight";

export default evolve({
  isPanelVisible: not,
  items: mergeDeepRight,
});
```

into this:

```js
import evolve from "ramda/src/evolve";
import isPanelVisible from "ramda/src/not";
import items from "ramda/src/mergeDeepRight";

export default evolve({
  isPanelVisible,
  items,
});
```

I think aliasing generic utility functions like this is a bad practice. I would like for plugin-apply-shorthand-properties to ignore cases like this.

**Comment by coderaiser** [comment]
Thank you :), fixed with fcb4c5d. Landed in `@putout/plugin-apply-shorthand properties v3.0.1` 🎉 . Is it works for you?

**Comment by EvgenyOrekhov** [comment]
Yes, it works, thank you!
